_data/people_list.yml

```
@@ -265,6 +258,11 @@
   website: https://rprebet.github.io/
   status: active
 
+- name: RPaegelow
+  email: raphael.paegelow@umontpellier.fr
+  github: RaphaelPaegelow
+  status: active
```

```suggestion
- name: Raphaël Paegelow
  affiliation: University of Lille
  email: raphael.paegelow@univ-lille.fr
  github: RaphaelPaegelow
  website: https://paegelow.fr/en/
  status: active
```
This is some more up to date information from their website